winter_credit/README.md

```diff
@@ -0,0 +1,74 @@
+# Description
+
+This module is designed to manage the specifics of the winter credit events.
+
+# Values
+
+TODO: Describe the meaning of the values returned by the module.
+
+# Lexicon
+
+
+The following are the terms in use in this module and coming as much as possible from the official Hydro-Quebec electricy rates document (page 31)
+https://www.hydroquebec.com/data/documents-donnees/pdf/electricity-rates.pdf
+
+## Period
+
+A period is time window where a specific billing or algorythmic logic is applied by HQ
+
+### normal
+A period when nothing special is defined by HQ rate policies
+
+### peak (peak hours)
+
+In this module: All hours from 06:00 to 09:00 and from 16:00 to 20:00 during the winter.
+
+This is when the critical peak events from hydro are happening. 
+
+In hydro's document there are also exclusions for specific holiday dates (Christmans, New year, Good Friday and Easter Monday) that we don't take into account here (yet)
+
+### temperature adjustement
+
+> I am unsure on how to name this. I thought reference made sense but in HQ's doc it means something completely different (see below)
+
+This period starts 5 hours before the next peak event's start time and has a duration of 3 hours. With the current peak period (as described above) it results in the following time periods: 
```

Comment:
Anciennement période de référence.

Ce [document](publicsde.regie-energie.qc.ca/projets/469/DocPrj/R-4057-2018-B-0062-DDR-RepDDR-2018_10_26.pdf#page=125) de la régie de l'énergie fait aussi référence à une période d'ancrage pour nommé cette période.

Reply:
I switched temperature adjustement to anchor period. It is a simpler term and probably less misleading than temperature adjustement.